specification/azurelargeinstance/resource-manager/Microsoft.AzureLargeInstance/preview/2023-07-20-preview/azurelargeinstance.json

```diff
@@ -0,0 +1,1312 @@
+        },
+        "osDisks": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Disk"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ],
+          "description": "Specifies information about the operating system disk used by Azure Large Instance."
+        }
+      },
+      "description": "Specifies the storage settings for the Azure Large Instance disks."
+    },
+    "OSProfile": {
+      "type": "object",
+      "properties": {
+        "computerName": {
+          "type": "string",
+          "description": "Specifies the host OS name of the Azure Large Instance."
+        },
+        "osType": {
+          "readOnly": true,
+          "type": "string",
+          "description": "This property allows you to specify the type of the OS."
+        },
+        "version": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Specifies version of operating system."
+        },
+        "sshPublicKey": {
+          "type": "string",
+          "description": "Specifies the SSH public key used to access the operating system."
+        }
+      },
+      "description": "Specifies the operating system settings for the Azure Large Instance."
+    },
+    "NetworkProfile": {
+      "type": "object",
+      "properties": {
+        "networkInterfaces": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IpAddress"
+          },
+          "x-ms-identifiers": [
+            "ipAddress"
+          ],
+          "description": "Specifies the network interfaces for the Azure Large Instance."
+        },
+        "circuitId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Specifies the circuit id for connecting to express route."
+        }
+      },
+      "description": "Specifies the network settings for the Azure Large Instance disks."
+    },
+    "IpAddress": {
+      "type": "object",
+      "properties": {
+        "ipAddress": {
+          "type": "string",
+          "description": "Specifies the IP address of the network interface."
+        }
+      },
+      "description": "Specifies the IP address of the network interface."
+    },
+    "AzureLargeStorageInstanceListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AzureLargeStorageInstance"
+          },
+          "description": "The list of AzureLargeStorage instances.",
+          "x-ms-identifiers": []
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of AzureLargeStorage instances."
+        }
+      },
+      "description": "The response from the Get AzureLargeStorageInstances operation."
+    },
+    "AzureLargeStorageInstance": {
+      "type": "object",
+      "x-ms-azure-resource": true,
```

Comment:
you shouldn't need to explicitly specify these bits, if you inherit v5 common types tracked resource using 'allOf':
-x-ms-azure-resource
-systemData